tests/unit/test_update_modes.py

```diff
@@ -12,14 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 import logging
-from tempfile import mkdtemp
 import os
+from tempfile import mkdtemp
```

Comment:
pytest actually has a built-in fixture for handling temporary files and directories https://docs.pytest.org/en/latest/tmpdir.html